screens/product/product.css

```diff
@@ -0,0 +1,112 @@
+@import url("https://fluid-ui-component.vercel.app/docs/docs.css");
+@import "../../style.css";
+.main-content {
+  display: flex;
+  margin-bottom: 3rem;
+}
+.page-header {
+  background-color: var(--white-color);
+  height: 10rem;
+  letter-spacing: 2px;
+  font-size: 2rem;
+  padding: 1rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.price-range {
+  display: flex;
+  justify-content: space-between;
+}
+.price-range-slider {
+  display: block;
+}
+.text-end {
+  text-align: end;
+}
+.sidebar {
+  background-color: var(--white-color);
+  width: 20%;
+  position: relative;
+}
+.main-heading {
+  margin: 0.1rem;
+  width: 80%;
+}
+
+.sidebar-heading {
+  font-weight: bold;
+  letter-spacing: 2px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.radio input {
+  margin-right: 0.3rem;
+}
+.checkbox input {
+  margin-right: 0.3rem;
+}
+.product-content {
+  width: 75%;
+  padding: 2rem;
+  display: flex;
+  justify-content: center;
+  gap: 2rem;
+  flex-wrap: wrap;
+}
+.card-container-portrait {
+  height: max-content;
+  position: relative;
+  width: 25%;
+  box-shadow: none;
+  padding: 1rem;
+  border: 1px solid var(--gray-color);
+  border-radius: 0;
+}
+.image-container {
+  padding: 1rem;
+  margin: 0%;
+  background-color: var(--white-color);
+}
+.card-container-portrait:hover {
+  border: 1px solid var(--black-color);
+  transition: 0.3s border ease-in-out;
+}
+.bottom-responsive-button {
+  display: none;
+}
+.card-price,
+.card-title {
+  margin: 0 1rem;
+}
+.price {
+  width: 80%;
+}
+@media only screen and (max-width: 500px) {
+  .sidebar {
+    display: none;
+  }
+  .product-content {
+    width: 90%;
+    margin: auto;
+  }
+  .card-container-portrait {
+    width: 90%;
+  }
+  .bottom-responsive-button {
+    justify-content: center;
+    letter-spacing: 2px;
+    display: flex;
+    font-size: 2rem;
+    width: 100%;
+    position: fixed;
+    z-index: 1;
+    bottom: 0;
+    border-top: 2px solid black;
+    margin-top: 1rem;
+    padding: 1rem 0;
```

Comment:
since `1rem` is used in many places, it is better to store it in a variable and reuse

Reply:
would keep this in mind